nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/reporting/ReportLineageToAtlas.java

```diff
@@ -640,7 +640,7 @@ private void consumeNiFiProvenanceEvents(ReportingContext context, NiFiFlow nifi
         final AnalysisContext analysisContext = new StandardAnalysisContext(nifiFlow, clusterResolvers,
                 // FIXME: This class cast shouldn't be necessary to query lineage. Possible refactor target in next major update.
                 (ProvenanceRepository)eventAccess.getProvenanceRepository());
-        consumer.consumeEvents(eventAccess, context.getStateManager(), events -> {
+        consumer.consumeEvents(null, eventAccess, context.getStateManager(), events -> {
```

Comment:
It would be more useful if we add another consumeEvents method signature (or replace existing one) having the last argument as `BiConsumer<ComponentMapHolder, List<ProvenanceEventRecord>>`.

Reply:
Agreed, I will update